tests/zoomus/components/webinar/test_list_panelists.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+import unittest
+
+from zoomus import components
+import responses
+
+
+def suite():
+    """Define all the tests of the module."""
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.makeSuite(ListPanelistsV2TestCase))
+    return suite
+
+
+class ListPanelistsV2TestCase(unittest.TestCase):
+    def setUp(self):
+        self.component = components.webinar.WebinarComponentV2(
+            base_uri="http://foo.com", config={"api_key": "KEY", "api_secret": "SECRET"}
+        )
+
```

Comment:
@chirag200666 Looks like you are missing tests here. Can I ask you to add some in? Thanks!

Reply:
added